DEVTENO/velonexa-frontend
# Feature/home

Title
testing masdian 

Description
This PR add navigation , it includes:
home
reels
explore
search
profile
notification

Changes
-

How to Test
Clone this branch (feature/sidebar)
Run npm install to install dependencies
Start the application with npm run dev
open localhost

Checklist
[x] I have completed the assigned task
[x] My code follows the code style of this project
[x] I have performed a self-review of my code
[x] I have commented my code, particularly in hard-to-understand areas

Additional Notes
please review my follow and unfollow implementation to see if it meets the given criteria
